packages/shared/components/AssetIcon.svelte

```diff
@@ -2,41 +2,36 @@
     import { AnimationRenderer } from '@auxiliary/animation'
     import { Icon as IconEnum, NETWORK_ICON_SVG } from '@auxiliary/icon'
     import { getIconColorFromString } from '@core/account'
-    import { COIN_TYPE, IPersistedNetwork, NetworkId } from '@core/network'
-    import { activeProfile } from '@core/profile'
+    import { COIN_TYPE, NetworkId } from '@core/network'
     import { isBright } from '@core/utils'
     import { ANIMATED_TOKEN_IDS, getAssetInitials, IPersistedAsset, TokenStandard } from '@core/wallet'
     import { Animation, Icon, VerificationBadge } from 'shared/components'
 
     export let asset: IPersistedAsset
-    export let large = false
-    export let small = false
+    export let large: boolean = false
+    export let small: boolean = false
```

Comment:
Wouldn't it be better to make this a `size` prop to avoid sending both of these at the same time?